renderdoc/driver/gl/egl_hooks.cpp

```diff
@@ -113,10 +114,14 @@ class EGLHook : LibraryHook
 
       bool isYFlipped = IsYFlipped(display, draw);
 
+      int multiSamples = 1;
+      EGL.GetConfigAttrib(display, config, EGL_SAMPLES, &multiSamples);
```

Comment:
I think this is fine but can you more strictly validate the value that comes back? Say only allow values of 1, 2, 4 or 8. That will protect against drivers returning garbage.